addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2386,6 +2386,9 @@ export class OdooEditor extends EventTarget {
                 this._selectTableCells(range);
                 appliedCustomSelection = true;
             }
+        } else if (selectedCells.length === 1){
+            this._selectTableCells(range);
+            appliedCustomSelection = true;
```

Comment:
@visp-odoo i think the correct line where it should be fixed is in this [line](https://github.com/odoo/odoo/pull/131161/files?diff=unified&w=0#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR2340) where we are handling 2 cell selection, Also since we are now focusing on single cell selection i think we dont need to deSelect the table in [line](https://github.com/odoo/odoo/pull/131161/files?diff=unified&w=0#diff-c309d19284e2e1f38a1dba1d68e0262371686595f3668a3bc0f1ca35e3189a0bR2313) @dmo-odoo what is your input on it, as deselecting the table will result in removal of toolbar.

Reply:
@shsa-odoo @visp-odoo It don't think I understand the problem here, so I don't understand the question either.

Reply:
@dmo-odoo there is `deselectTable()` method is called in `  _handleSelectionInTable` @shsa-odoo wants me to remove it because he thinks that if we already set selection on table then why we need to remove that selection.

IMO we can't remove `deselectTable()`. We need your input on this whether can we remove it or not ?

Reply:
I summon @Zinston as I cannot reply to this question without actually investigating further.

Reply:
@Zinston here, at your service :vulcan_salute:
Long story short, no you can't remove it. We need to first deselect any table that is already selected before making the new table selection (if any) because we're not sure we're going to select the same cells as before if we're even going to select any cell at all. See this as a refresh of the rendering: erase everything then redraw only what you need.

Reply:
@Zinston here, at your service :vulcan_salute:
@dmo-odoo @visp-odoo @shsa-odoo 
Long story short, no you can't remove it. We need to first deselect any table that is already selected before making the new table selection (if any) because we're not sure we're going to select the same cells as before if we're even going to select any cell at all. See this as a refresh of the rendering: erase everything then redraw only what you need.